frontend/elements/src/entry/jig/publish/page-2.ts

```diff
@@ -88,11 +86,11 @@ export class _ extends LitElement {
     const STR_SUBTITLE ="What would you like to do next?"
 
     return html`    
-    <main>
+    <main style="background-image:url('${mediaUi("Background@2x.jpg")}')">
         <div class="wrapper">
             
             <div class="inside-wrapper">
-                <slot name="animation"></slot>
+            <placeholder-img class="animation"></placeholder-img>
```

Comment:
"placeholder" is an awkward name for this - hard to tell what it's supposed to do by the name... and convention for our images is generally `img-*`